### DISCUSSION TITLE
Flickering along edges and decals (is back) – Identified on Mac Chrome

### DISCUSSION THREAD

**Discussion by phreak** [google/model-viewer#4957] [General]
With the last version of Chrome on MacOS the "Flickering" in of GLBs in MV (we had some versions ago) is back.

Edges as well as decals are flickering on manual rotation and auto-rotate for the blink of an eye, until it seems, the shader rendering "closes the edge" to it's natural color. Especially on auto-rotate it's really bad as the models seem to flicker sometimes on many edges and decals.

_Chrome 131.0.6778.86 (Official Build) (arm64)_

It's not appearing on Firefox or Safari for Mac. Problem appears in the official online viewer (https://modelviewer.dev/editor/) and in custom MV's.

Can anyone confirm this with their own models and provide more technical information so @elalish could probably address this with the Chrome/Webkit team (if it's in their domain)?

Screen recording below:
https://github.com/user-attachments/assets/d2678343-287b-4105-b568-f2e172a2f019

**Comment by elalish** [comment]
Thanks for the video, that's ugly! I've never seen that particular issue before, and I don't see it now on my Macbook Pro M1 with the latest Chrome. Is it particular to this model? Can you give a link where we can repro?
